src/ResourceManager/HDInsight/Commands.HDInsight/ManagementCommands/AddAzureHDInsightConfigValuesCommand.cs

```diff
@@ -82,6 +82,18 @@ public class AddAzureHDInsightConfigValuesCommand : HDInsightCmdletBase
         [Parameter(HelpMessage = "Gets the RServer configurations.")]
         public Hashtable RServer { get; set; }
 
+        [Parameter(HelpMessage = "Gets the Spark Defaults configurations of this HDInsight cluster.")]
+        public Hashtable SparkDefaults { get; set; }
```

Comment:
@jeffersonezra out of curiosity, is it possible for a configuration to have values for both Spark and Spark2 properties? For example, should it be valid for a user to provide values for `SparkDefaults` and `Spark2Defaults`?